tests/Unit/Evolution/DiscontinuousGalerkin/Test_ConservationHelper.cpp

```diff
@@ -0,0 +1,166 @@
+      }
+    }
+
+    if (static_cast<bool>(normal_dot_mesh_velocity)) {
+      get(*packaged_abs_char_speed) =
+          abs(volume_double * get(var1) - get(*normal_dot_mesh_velocity));
+    } else {
+      get(*packaged_abs_char_speed) = abs(volume_double * get(var1));
+    }
+    return max(get(*packaged_abs_char_speed));
+  }
+
+  void dg_boundary_terms(
+      const gsl::not_null<Scalar<DataVector>*> boundary_correction_var1,
+      const gsl::not_null<tnsr::i<DataVector, Dim, Frame::Inertial>*>
+          boundary_correction_var2,
+      const Scalar<DataVector>& var1_int,
+      const Scalar<DataVector>& normal_dot_flux_var1_int,
+      const tnsr::i<DataVector, Dim, Frame::Inertial>& var2_int,
+      const tnsr::i<DataVector, Dim, Frame::Inertial>& normal_dot_flux_var2_int,
+      const Scalar<DataVector>& abs_char_speed_int,
+      const Scalar<DataVector>& var1_ext,
+      const Scalar<DataVector>& normal_dot_flux_var1_ext,
+      const tnsr::i<DataVector, Dim, Frame::Inertial>& var2_ext,
+      const tnsr::i<DataVector, Dim, Frame::Inertial>& normal_dot_flux_var2_ext,
+      const Scalar<DataVector>& abs_char_speed_ext,
+      const dg::Formulation dg_formulation) const noexcept {
+    if (dg_formulation == dg::Formulation::WeakInertial) {
+      get(*boundary_correction_var1) =
+          0.5 *
+              (get(normal_dot_flux_var1_int) - get(normal_dot_flux_var1_ext)) -
+          0.5 * max(get(abs_char_speed_int), get(abs_char_speed_ext)) *
+              (get(var1_ext) - get(var1_int));
+      for (size_t i = 0; i < Dim; ++i) {
+        boundary_correction_var2->get(i) =
+            0.5 * (normal_dot_flux_var2_int.get(i) -
+                   normal_dot_flux_var2_ext.get(i)) -
+            0.5 * max(get(abs_char_speed_int), get(abs_char_speed_ext)) *
+                (var2_ext.get(i) - var2_int.get(i));
+      }
+    } else {
+      get(*boundary_correction_var1) =
+          -0.5 *
+              (get(normal_dot_flux_var1_int) + get(normal_dot_flux_var1_ext)) -
+          0.5 * max(get(abs_char_speed_int), get(abs_char_speed_ext)) *
+              (get(var1_ext) - get(var1_int));
+      for (size_t i = 0; i < Dim; ++i) {
+        boundary_correction_var2->get(i) =
+            -0.5 * (normal_dot_flux_var2_int.get(i) +
+                    normal_dot_flux_var2_ext.get(i)) -
+            0.5 * max(get(abs_char_speed_int), get(abs_char_speed_ext)) *
+                (var2_ext.get(i) - var2_int.get(i));
+      }
+    }
+  }
+};
+
+template <size_t Dim>
+void test() {
+  const Correction<Dim> correction{};
+  const Mesh<Dim - 1> face_mesh{Dim * Dim, Spectral::Basis::Legendre,
+                                Spectral::Quadrature::Gauss};
+  TestHelpers::evolution::dg::test_boundary_correction_conservation<
+      System<Dim>>(correction, face_mesh,
+                   tuples::TaggedTuple<Tags::VolumeDouble>{2.3});
+}
+}  // namespace
+
+SPECTRE_TEST_CASE("Unit.Evolution.DG.ConservationHelper", "[Unit][Evolution]") {
```

Comment:
To follow our naming patterns, I suggest `Unit.Evolution.DG.BoundaryCorrections(Helper?).cpp`